packages/webpack-build-scripts/externals.js

```diff
@@ -16,11 +16,14 @@ module.exports = externalize({
     "jquery": "$",
     "moment": "moment",
     "moment-timezone": "moment",
+    "popper.js": "Popper",
     "react": "React",
-    "react-transition-group": "ReactTransitionGroup",
     "react-day-picker": "DayPicker",
     "react-dom": "ReactDOM",
-    "tslib": "tslib",
+    "react-popper": "ReactPopper",
+    "react-transition-group": "ReactTransitionGroup",
+    "resize-observer-polyfill": "ResizeObserver",
```

Comment:
this feels a little silly to require as an external... but I guess it's consistent with dom4

Reply:
yah i could easily be convinced to remove this line and bundle this one lib, but might as well be consistent